src/pages/admin/AdminUsersPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { listInstitutionUsers } from '../../api/userApi';
+import Heading from '../../components/Heading';
+import SubHeading from '../../components/SubHeading';
+import { useSelector } from 'react-redux';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import { UserAdmin, RoleName } from '../../types/user.types';
+import styled from 'styled-components';
+import Card from '../../components/Card';
+import { Button, FormControlLabel, Checkbox } from '@material-ui/core';
+import UserList from './UserList';
+import NormalText from './../../components/NormalText';
+
+const StyledButton = styled(Button)`
+  margin-top: 1rem;
+`;
+
+const StyledContainer = styled.div`
+  margin-bottom: 2rem;
+`;
+
+const AdminUsersPage: FC = () => {
+  const { t } = useTranslation('profile');
+  const user = useSelector((store: RootStore) => store.user);
+  const [userList, setUserList] = useState<UserAdmin[]>([]);
+  const [autoAssignCreators, setAutoAssignCreators] = useState(true);
+
+  useEffect(() => {
+    const getUsers = async () => {
+      const users = await listInstitutionUsers(user.institution);
+      // TODO backend
+      // if (users?.error) {
+      //   dispatch(addNotification(t('feedback:error.get_publications'), 'error'));
+      // } else {
+      setUserList(users);
+      // }
+    };
+
+    getUsers();
+  }, [user.institution]);
+
+  const handleCheckAutoAssignCreators = () => {
+    setAutoAssignCreators(!autoAssignCreators);
+  };
+
+  return (
+    <Card>
+      <Heading>{t('organization.user_administration')}</Heading>
+      <SubHeading>{t('roles.institution_admins')}</SubHeading>
+      <StyledContainer>
+        {userList.filter(user => user.roles.find(role => role === RoleName.ADMIN)) && (
+          <UserList userList={userList.filter(user => user.roles.find(role => role === RoleName.ADMIN))} />
+        )}
```

Comment:
blir ikke dette det samme?
```
{userList.filter(user => user.roles.some(role => role === RoleName.ADMIN)).map(adminUsers =>
     <UserList userList={adminUsers} />
)}
```

Reply:
samme for curator og editor

Reply:
du har helt sikkert rett

Reply:
Trenger å sende en array av users inn i UserList så det virker ikke allikevel

Reply:
trekker det ut i en funksjon for å forenkle

Reply:
aah, riktig. nice!